### DISCUSSION TITLE
Anyway to install UniGetUI without UAC headless?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [Devolutions/UniGetUI#2520] [Q&A] [answered]
I tried installing with the PowerShell WinGet module on a machine I want it installed only in the user's scope:

```powershell
Find-WinGetPackage -Source winget -Moniker unigetui | Install-WinGetPackage -Scope User
```
The installer installed in "C:\Program Files..."

I searched around, but was unsure of whether this should be filed as a feature requiest, or if this is something I'm doing wrong.

**Comment by marticliment** [comment, marked as answer]
You should override the installer arguments. More info here: https://github.com/marticliment/UniGetUI?tab=readme-ov-file#command-line-parameters

**Reply by Lockszmith-GH** [reply]
Finally did it on a brand new Windows 11 machine, so I know it's safe to share:

ℹ️NOTE: Microsoft Visual C++ Redistributable is a requirement, so if you want to prep the system for zero friction for the user install, run this first as admin:
```powershell
winget.exe install --id "Microsoft.VCRedist.2015+.x64" --exact --source winget --accept-source-agreements
#or with the powershell module:
Find-WinGetPackage -MatchOption Equals -Id 'Microsoft.VCRedist.2015+.x64' | Install-WinGetPackage -Scope User
```

Then Installing WinGetUI in user context:
```powershell
winget install SomePythonThings.WingetUIStore --exact --source winget --accept-package-agreements --accept-source-agreements --custom "/CURRENTUSER"
#or with the powershell module:
Find-WinGetPackage -MatchOption Equals -Id 'SomePythonThings.WingetUIStore' | Install-WinGetPackage -Scope User -Custom '/CURRENTUSER'
```
